client/state/ui/action-log/reducer.js

```diff
@@ -22,7 +22,7 @@ const relevantTypes = {
 	EDITOR_PASTE_EVENT,
 	FIRST_VIEW_HIDE,
 	GUIDED_TOUR_UPDATE,
-	THEMES_RECEIVE,
+	THEMES_REQUEST_SUCCESS,
```

Comment:
@mcsf is this an appropriate change now that 'THEMES_REQUEST_SUCCESS` is the sole signifier of themes arriving at the client?